packages/node_modules/@node-red/nodes/core/function/89-delay.js

```diff
@@ -140,8 +141,9 @@ module.exports = function(RED) {
                 var id = ourTimeout(function() {
                     node.idList.splice(node.idList.indexOf(id),1);
                     if (node.idList.length === 0) { node.status({}); }
-                    node.send(msg);
-                }, delayvar);
+                    send(msg);
+                    done();
+                }, delayvar, () => done("queue cleared"));
```

Comment:
I don't think clearing the delay queue should be considered an error - which calling `done(...)` with an argument suggests. The queue is being cleared because the user has asked for that to happen.

(This applies to all the places you call `done(...)` with a string argument).

Reply:
ok.  I'll remove an argument on done(...) when clearing the queue:
- https://github.com/node-red/node-red/blob/ee68fe063be913ac4754295882e537f8c537eb34/packages/node_modules/%40node-red/nodes/core/function/89-delay.js#L124
- https://github.com/node-red/node-red/blob/ee68fe063be913ac4754295882e537f8c537eb34/packages/node_modules/%40node-red/nodes/core/function/89-delay.js#L146
- https://github.com/node-red/node-red/blob/ee68fe063be913ac4754295882e537f8c537eb34/packages/node_modules/%40node-red/nodes/core/function/89-delay.js#L223
- https://github.com/node-red/node-red/blob/ee68fe063be913ac4754295882e537f8c537eb34/packages/node_modules/%40node-red/nodes/core/function/89-delay.js#L266
- https://github.com/node-red/node-red/blob/ee68fe063be913ac4754295882e537f8c537eb34/packages/node_modules/%40node-red/nodes/core/function/89-delay.js#L284